src/include/rados/librados.h

```diff
@@ -15,6 +15,10 @@
 #ifndef CEPH_LIBRADOS_H
 #define CEPH_LIBRADOS_H
 
+#ifdef _WIN32
+#include "../windows/windefs.h"
+#endif
+
```

Comment:
Perhaps put this in `compat.h`, and include that here.

Reply:
Sure.

Reply:
Actually, we can't use compat.h here as it includes #include "acconfig.h", which would break external projects using librbd (e.g. the Python bindings).

Reply:
makes sense